tests/userController.test.js

```diff
@@ -100,4 +100,53 @@ describe("user controller", () => {
       });
     });
   });
+
+  describe("gets a user", () => {
+    it("gets an existing user", async() => {
+      const req = {
+        params: { id: 1}
+      };
+      const res = {
```

Comment:
Redundant. You already defined this on the before each at the parent describe block above.

Reply:
I have removed all of the redundant code